cmd/vic-machine/create/create.go

```diff
@@ -41,7 +41,7 @@ const (
 	// Max permitted length of Virtual Switch name
 	MaxDisplayNameLen = 31
 	// Default debug level in create
-	DefaultDebug = 1
+	DefaultDebug = -1
```

Comment:
might move this definition to this file, and update following line to ignore this default value:
https://github.com/vmware/vic/blob/master/lib/install/validate/config_to_data.go#L217
So that we'll not show the -1 as one configuration option in vic-machine inspect config.
The usage message didn't show -1 as a valid value, this is to avoid confusion.